tests/unit/src/test/scala/tests/RenameLspSuite.scala

```diff
@@ -515,7 +516,7 @@ class RenameLspSuite extends BaseLspSuite("rename") {
   )
 
   def renamed(
-      name: String,
+      name: TestOptions,
```

Comment:
This change doesn't seem neccessary. Why use TestOptions here?

Reply:
For testing to be able to add ".only" at the end of single test. It only applied to TestOptions.